packages/daemon/src/daemon.js

```diff
@@ -445,13 +445,12 @@ const makeEndoBootstrap = (
       const workerFormulaIdentifier = `worker-id512:${formulaNumber}`;
       // Behold, recursion:
       // eslint-disable-next-line no-use-before-define
-      const external = makeIdentifiedHost(
+      return makeIdentifiedHost(
```

Comment:
How did this pass static type-checking previously? I guess type coverage isn't thorough?

Reply:
I’m betting we have to disable implicit any to get more juice from ts here.